frontend/src/constants.js

```diff
@@ -19,3 +19,26 @@ export const CredentialTypes = Object.freeze({
     label: "Unknown",
   },
 });
+
+/** API Route paths */
+export const ApiRoutes = Object.freeze({
+  ISSUER: '/issuer'
+});
+
+export const CredentialExchangeRoles = Object.freeze( {
+  ISSUER: 'issuer',
+  HOLDER: 'holder'
+});
+
+export const CredentialExchangeStates = Object.freeze( {
```

Comment:
will it be easier if we make this also upper case from the backend side? see comment above?

Reply:
done.